Tests/administrator/test_working.py

```diff
@@ -0,0 +1,42 @@
+# Third Party Imports
+from selenium import webdriver
+
+# Django imports
+from django.contrib.staticfiles.testing import LiveServerTestCase
+
+
+# Local Project Imports
+from selenium.webdriver.common.keys import Keys
+
+
+class DummyTesting(LiveServerTestCase):
+    """
+    Dummy Test Class to check the tests are working correctly.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+
+        cls.driver = webdriver.Firefox()
+        cls.driver.implicitly_wait(5)
+        cls.driver.maximize_window()
+        super(DummyTesting, cls).setUpClass()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.driver.quit()
+        super(DummyTesting, cls).tearDownClass()
+
+    def test_working(self):
+        """
+        Dummy Test function to check working of selenium
+        Delete this function after the first test for this
+        Class is added.
+        """
+        self.driver.get("http://www.python.org")
+        self.assertIn('Python', self.driver.title)
+        element = self.driver.find_element_by_name('q')
+        element.clear()
+        element.send_keys('pycon')
+        element.send_keys(Keys.RETURN)
+        self.assertNotIn('No results found.', self.driver.page_source)
```

Comment:
No new line at the end in all the files.

Reply:
Yes it's recommended by pep8, due to consistency in every line being terminated by '\n'.

Reply:
Yes I follow that rule, here is the screenshot of the file in pycharm which shows that there is a newline at end.
![newline](https://user-images.githubusercontent.com/21009189/40196460-3914d984-5a2e-11e8-9841-bf1ea4af0f38.png)

Reply:
Can you check if some other IDE/editor shows the newline too? Something like Sublime, VSCode, etc.?

Reply:
Okay I'll check it, Thanks.